lld/test/ELF/gc-sections-with-provide.s

```diff
@@ -0,0 +1,60 @@
+# REQUIRES: x86
+
+# This test verifies that garbage-collection is correctly garbage collecting
+# unused sections when the symbol of the unused section is only referred by
+# an unused PROVIDE symbol.
+
+# RUN: rm -rf %t && split-file %s %t && cd %t
+# RUN: llvm-mc -filetype=obj -triple=x86_64 a.s -o a.o
+# RUN: ld.lld -o a_nogc a.o -T script.t
+# RUN: llvm-readobj --sections --symbols a_nogc | FileCheck -check-prefix=NOGC %s
```

Comment:
`-S` (--sections) is unused

Reply:
I have changed the test to use `llvm-nm` based on your suggestion in another comment.